sdk/regionmove/Microsoft.Azure.Management.Migrate/AzSdk.RP.props

```diff
@@ -0,0 +1,7 @@
+<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <!--This file and it's contents are updated at build time moving or editing might result in build failure. Take due deligence while editing this file-->
+  <PropertyGroup>
+    <AzureApiTag>AzureResourceMover_2019-10-01-preview;</AzureApiTag>
+    <PackageTags>$(PackageTags);$(CommonTags);$(AzureApiTag);</PackageTags>
+  </PropertyGroup>
+</Project>
```

Comment:
Can you do me a favor and make another commit to add a line return here? I merged the pipeline change after your commit. Running `check-enforcer /reset` didn't seem to retrigger it. Another commit may help run it normally. Thanks

Reply:
Done